src/screens/WalletType.js

```diff
@@ -118,6 +119,27 @@ class WalletType extends React.Component {
 
       wallet.startWallet(null, '', null, '', this.props.history, false, xpub);
       hathorLib.wallet.markBackupAsDone();
+
+      const tokenSignatures = hathorLib.storage.getItem('wallet:token:signatures');
```

Comment:
Maybe we could keep this logic in a utils file (wallet or ledger).

But that's just a suggestion.